### PR TITLE
Feature/lohrd/j browse

### PR DESCRIPTION
**Pull request by lohrd**
This pull request addresses issue #25 

- Add Jbrowse route under '/jbrowse'
- Need to add ability to update BAM viewed from SRA page via prop injection into Jbrowse page
- Also, maybe add ability to generate a 'cart' of different bam files to load each one into Jbrowse for cross comparisons

Live Preview: https://feature-lohrd-jbrowse.d1w6d75be7tofa.amplifyapp.com

### CODE REVIEW

**Comment by ababaian** [comment]
Bam files are about 40% complete for the upgrade. Should be ready for an update in a day or two.

File: public/img/cds.svg [hunk 1]
```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (victorlin) [inline public/img/cds.svg:1]: why are there so many new additions under `public/img`? shouldn't the `@gmod/jbrowse` package take care of things like this?

File: src/pages/Jbrowse.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+const App = () => {
+	const features = [];
```
Comment (victorlin) [inline src/pages/Jbrowse.jsx:4]: tabs -> spaces

File: src/components/Paginator.jsx [hunk 3]
```diff
@@ -45,10 +45,10 @@ const Paginator = ( {pageNumber, setPageNumber, numberOfPages, getNumberOfPages}
 
   return (
     <div className={centerButtons}>
-      {pageNumber == 1 ? 
+      {pageNumber === 1 ?
```
Comment (victorlin) [inline src/components/Paginator.jsx:48]: looks like changes to this file are causing merge conflicts

File: public/img/cds.svg (hunk 1 above)
Comment (lohrd) [reply public/img/cds.svg:1]: The @gmod/jbrowse package doesn't have actual npm features and support yet, so all of the files have to been manually inserted in the root in an img folder, source: https://github.com/cmdcolin/jbrowse_in_react_app/ & from the dev's comment on our issue's page "The @gmod/jbrowse-web npm package is non functional right now, but is our work in progress jbrowse 2 release. Docs for jbrowse 2 are starting to come together at http://jbrowse.org/jb2/"

I know this is less than ideal, but until the package is up and running I think we can live with it, yeah?

**Review by victorlin — APPROVED** [review]

**Comment by ababaian** [comment]
For `GenBank` search, can the field also contain the 'name' of the sequence in the search space not just the accession?

**Pull request merged**